tasks/pkg-redhat.yml

```diff
@@ -64,12 +64,13 @@
         dest: /tmp/DATADOG_RPM_KEY_E09422B3.public
```

Comment:
I can't comment on line 61, but we might want to stop referring to a key expired 2 years ago as `new` :sweat_smile: